beacon-chain/attestation/service.go

```diff
@@ -65,21 +65,6 @@ func (a *Service) IncomingAttestationFeed() *event.Feed {
 	return a.incomingFeed
 }
 
-// ContainsAttestation checks if an attestation has already been received and aggregated.
-func (a *Service) ContainsAttestation(bitfield []byte, hash [32]byte) (bool, error) {
```

Comment:
I split this into 2 functions:

1. `beaconDB.getAttestation(hash)`
2. `attestation.containsValidator(bitfield)`